xlml/apis/metric_config.py

```diff
@@ -94,3 +97,4 @@ class MetricConfig:
   json_lines: Optional[JSONLinesConfig] = None
   tensorboard_summary: Optional[SummaryConfig] = None
   profile: Optional[ProfileConfig] = None
+  write_results_to_bigquery: bool = False
```

Comment:
What's the purpose of this flag?

Reply:
This flag allows a dag to override the restriction for writing to BigQuery. If this flag is set true then we will allow the run to write to BigQuery.